src/clr-angular/data/datagrid/all.spec.ts

```diff
@@ -83,18 +88,19 @@ describe('Datagrid', function() {
     DatagridColumnToggleSpecs();
     DatagridColumnToggleButtonSpecs();
     DatagridHideableColumnDirectiveSpec();
+    WrappedCellSpec();
+    WrappedColumnSpec();
+    WrappedRowSpec();
+    // DatagridProjectionSpec();
```

Comment:
Should this be on or off?

Reply:
Merge artifact will remove as this suite was replaced by the three `Wrapped*Spec`'s
Might have ben a rebase artifact. Will remove.